### PR TITLE
Closes AgileVentures/MetPlus_tracker#185

### PR DESCRIPTION
**Pull request by sidhene**
Ready to be reviewed

### CODE REVIEW

File: app/controllers/agency_people_controller.rb [hunk 1]
```diff
@@ -60,6 +60,28 @@ def update
...
   end
 
+  def edit_profile
+    @agency_person = AgencyPerson.find(params[:id])
+  end
+
+  def update_profile
+    @agency_person = AgencyPerson.find(params[:id])
+    person_params = agency_person_params
+    if person_params['password'].to_s.length == 0
```
Comment (joaopapereira) [inline app/controllers/agency_people_controller.rb]: We should create a Issue to refactor this somewhere because I have seen this code in multiple places

**Comment by joaopapereira** [comment]
Looks good.
Waiting on the input of someone else

**Comment by patmbolger** [comment]
I have reviewed the code and it looks fine.  (agree with the refactor idea also)

**Comment by joaopapereira** [comment]
Going to merge the issue

**Pull request merged**